dropwizard-jersey/src/test/java/io/dropwizard/jersey/DropwizardResourceConfigTest.java

```diff
@@ -74,6 +76,23 @@ public void logsEndpoints() {
                 .contains("GET     /another (io.dropwizard.jersey.DropwizardResourceConfigTest.ImplementingResource)");
     }
 
+    @Test
+    public void logsEndpointsSorted() {
+        rc.register(DummyResource.class);
+        rc.register(TestResource2.class);
+        rc.register(TestResource.class);
+        rc.register(ImplementingResource.class);
+
+        assertThat(rc.getEndpointsInfo())
+                .contains("    GET     / (io.dropwizard.jersey.dummy.DummyResource)\n"
```

Comment:
This test will fail on Windows, because it has different line breaks (\r\n rather than \n).
Could we wrap an expected string in `String.format` or use a regular expression?